### ISSUE TITLE
open core

### ISSUE THREAD

**Issue by pdurbin** [good-labs/greater-good-affirmation#24]
Does "open core" have anything to do with the Greater Good Affirmation? Is open core for the greater good? If open core is off topic here, perhaps it's on topic over at https://github.com/good-labs/FUNDING.md

**Comment by vsoch** [comment]
Yep, it was an original [bullet point](https://github.com/good-labs/greater-good-affirmation/commit/95bf62f72bc89aa38d066dc0e0ed7f890d66af2f#diff-0669a09379be7c4afa196810079dce7cL10) until we had that long discussion and updated the [points to this](https://github.com/good-labs/greater-good-affirmation/commit/95bf62f72bc89aa38d066dc0e0ed7f890d66af2f#diff-2e849ccde8aabaff877eedae5f4e7b6cR1).

**Comment by pdurbin** [comment]
Ah, great, I forgot that open core was in there at one point. Thanks!

**Comment by vsoch** [comment]
Should it be added back?

**Comment by pdurbin** [comment]
Maybe. I actually like all three of those bullets:

 - The majority of contributors represent dispersed community (and not required participants for a company).
 - The community strives for open communication for discussion around development.
 - Software is provided under an open source license, with no private distributions or versions with special privileges (no "open core" models).

I would maybe reword the first bullet a bit (I'm not sure how exactly), but I know what you're getting at. People are into it because they find the software useful, not because their boss told them to fix a bug or implement a feature.

As I've probably made evidently clear over in https://github.com/good-labs/slopi-communication I am in favor of open communication (the second bullet above). I especially like it when the roadmap can be discussed openly.

I feel a little weird and "judgy" (if that's a word) saying "if you're open core, you aren't for the greater good." So I'm hesitating. But I'm glad I opened this issue because I didn't remember open core was in there.

I doubt I'll make a snap decision on any of this. These are hard problems. It's hard to put into words what's in your heart.
